source/stylesheets/modules/_sticky.scss

```diff
@@ -0,0 +1,13 @@
+// Sticky Element Component
+//
+// Example Usage:
+//
+// <span class="header__title">
+//   My Product <span class="sticky">Beta</span>
+// </span>
+
+.sticky {
+  position: -webkit-sticky;
+  position: sticky;
```

Comment:
Can you have two properties on one class? I thought they would have overwritten each other.

Reply:
I think it depends on the browser. 
Safari requires the `position: -webkit-sticky`.

Reply:
Ah ok